### ISSUE TITLE
Change title of mpv for trackma

### ISSUE THREAD

**Issue by PhosCity** [pystardust/ani-cli#568]
**Is your feature request related to a problem? Please describe.**
First, I understand that you won't deal with anilist/mal sync but hear me out. The current title that is used in the script in the format `ani-cli:  $anime_id ep $episode` is not understood by trackma and therefore cannot automatically update the anime.

**Describe the solution you'd like**
This can be fixed by simply removing `ani-cli: ` from the title. If you must have ani-cli in the title then `[ani-cli]  $anime_id ep $episode` has higher chance of being recognized by trackma. Removing dash from `$anime_id` in title would be even better. Even though you guys won't receive feature requests for anime tracking, this constitutes of only changing 2 lines on your part, so consider changing the title of mpv.

**Comment by ura43** [comment]
This sounds reasonable. I use trackma myself and believe chokerman does too.
Would this enable automatic trackma updates for their cli interface too?

**Comment by PhosCity** [comment]
I only use cli interface of trackma so I can confirm that automatic updates work in cli interface. Furthermore, I tried in trackma-gtk and it works just the same.

But additional info: To make automatic update work in trackma, I changed the tracker to mpris in trackma preferences. By default, it was in local for me and it didn't work. You may also need to install mpv-mpris(manually or through package manager) and python-pydbus to make it work.

**Comment by iamchokerman** [comment]
@PhosCity i'm currently testing this out, on a brand new install, if i understood correctly the title that it should display in mpv should be smtg like "naruto ep 1" instead of the current "ani-cli: naruto ep 1" right? also, do you expect it to display "serial experiments lain ep 1" instead of "serial-experiments-lain ep 1" ? also, do you need a trackma instance to be open to get it to work? (feel free to join the discord, i might have some more questions in the future, it might simplify the process, but it's up to you ofc)

**Comment by PhosCity** [comment]
> @PhosCity i'm currently testing this out, on a brand new install, if i understood correctly the title that it should display in mpv should be smtg like "naruto ep 1" instead of the current "ani-cli: naruto ep 1" right? also, do you expect it to display "serial experiments lain ep 1" instead of "serial-experiments-lain ep 1" ? also, do you need a trackma instance to be open to get it to work? (feel free to join the discord, i might have some more questions in the future, it might simplify the process, but it's up to you ofc)

That is correct. The name I would like the title to be in is as follows (in the order to best to ok):
```Serial Experiments Lain - 1
serial experiments lain - 1
[ani-cli] serial experiments lain - 1
serial-experiments-lain -1```

The rationale behind this is any text in the title can be considered as the name of the anime by trackma. So when `ani-cli` is present in the title, instead of searching for anime named `naruto`, it's searching for an anime called `ani-cli naruto`. No wonder it doesn't find it. The same goes for having `ep` in the title but that can still be included if you want. The reason why `[ani-cli]` has a chance of being recognized is because in fansubbing, many fansub teams follow the convention of `[Fansub] Anime name` so trackma has accounted for such naming convention.

Also trackma must be running, either it's gui or the cli. 

I'll join the discord. My discord username is the same as my github username so if you have more questions feel free to ask questions there.

**Comment by iamchokerman** [comment]
> 

very well, thank you for the explanation, i'll try to look further into it, i've already done some testing but couldn't get it to work just yet. i saw u on discrod, thank you for that 👍
